### ISSUE TITLE
eel dependency will break in v1.0.0

### ISSUE THREAD

**Issue by egreglee** [georgegach/flowiz#8]
```
python -m flowiz.gui
> GUI webpath: /home/greg/miniconda3/envs/spynet/lib/python3.8/site-packages/flowiz/gui/web
Traceback (most recent call last):
  File "/home/greg/miniconda3/envs/spynet/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/greg/miniconda3/envs/spynet/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/greg/miniconda3/envs/spynet/lib/python3.8/site-packages/flowiz/gui/__main__.py", line 72, in <module>
    eel.start('index.html', options={
  File "/home/greg/miniconda3/envs/spynet/lib/python3.8/site-packages/eel/__init__.py", line 142, in start
    raise RuntimeError(api_error_message)
RuntimeError: 
----------------------------------------------------------------------------------
  'options' argument deprecated in v1.0.0, see https://github.com/ChrisKnott/Eel
  To suppress this error, add 'suppress_error=True' to start() call.
  This option will be removed in future versions
----------------------------------------------------------------------------------
```
flowiz was installed using pip:

```
pip install eel
pip install flowiz -U
```

**Comment by enric1994** [comment]
Same issue, tried installing via pip and source

**Comment by Roger-Gu** [comment]
Same issue...

**Comment by georgegach** [comment]
Fixed in #5 and added exact package versions in requirements for future-proofness
